source/PlayerInfo.cpp

```diff
@@ -2556,14 +2635,41 @@ void PlayerInfo::RegisterDerivedConditions()
 	unpaidFinesProvider.SetGetFunction([this](const string &name) { return min(limit, accounts.TotalDebt("Fine")); });
 
 	auto &&unpaidSalariesProvider = conditions.GetProviderNamed("unpaid salaries");
-	unpaidSalariesProvider.SetGetFunction([this](const string &name) { return min(limit, accounts.SalariesOwed()); });
+	unpaidSalariesProvider.SetGetFunction([this](const string &name) { return min(limit, accounts.CrewSalariesOwed()); });
 
 	auto &&unpaidMaintenanceProvider = conditions.GetProviderNamed("unpaid maintenance");
 	unpaidMaintenanceProvider.SetGetFunction([this](const string &name) { return min(limit, accounts.MaintenanceDue()); });
 
 	auto &&creditScoreProvider = conditions.GetProviderNamed("credit score");
 	creditScoreProvider.SetGetFunction([this](const string &name) { return accounts.CreditScore(); });
 
+	// Read/write assets and debts.
+	auto &&salaryIncomeProvider = conditions.GetProviderPrefixed("salary: ");
```

Comment:
[This comment](https://github.com/endless-sky/endless-sky/pull/7210/files#r962377603) from @Amazinite also applies here.

Reply:
Comment handled.